synapse/tests/utils.py

```diff
@@ -641,6 +602,10 @@ def thisHostMustNot(self, **props):  # pragma: no cover
     async def getTestDmon(self, mirror='dmontest'):
 
         with self.getTestDir(mirror=mirror) as dirn:
+
+            # Copy test certs
+            shutil.copytree(os.path.join(self.getTestFilePath(), 'certdir'), os.path.join(dirn, 'certs'))
```

Comment:
`shutil.copytree(self.getTestFilePath('certdir'), os.path.join(dirn, 'certs'))`

should work fine

Reply:
ok